### DISCUSSION TITLE
GIF support for tvOS

### DISCUSSION THREAD

**Discussion by harikrushna1993** [kean/Nuke#485] [Q&A] [answered]
Hi team,
i m using this library for my tvOS application where i want to add GIF, so can anyone please tell this library supports GIF automatically(like SDWebImage) or we need to do extra code to support GIF.

**Comment by kean** [comment, marked as answer]
[NukeUI](https://github.com/kean/NukeUI) supports GIF on tvOS out of the box. NukeUI, in general, contains everything that's related to rendering. I'm pushing the first tag later today, but it's still going to be a pre-release version.

As an alternative, you can use Gifu or any other package for GIF rendering. See ["Image Formats: GIF"](https://kean.blog/nuke/guides/image-formats#gif) for more info.

> By the way, thanks for opening a discussion and not an issue!
